From 0d06dfa564fc5e411bdb37d473005882f324cf8f Mon Sep 17 00:00:00 2001
From: Jarkko Nikula <jhnikula@gmail.com>
Date: Thu, 14 Apr 2011 16:21:58 +0300
Subject: [PATCH 34/48] OMAP2PLUS: cpufreq: Fix typo when attempting to set
 mpu_clk for OMAP4

Fix this typo as there is no dpll_mpu_ck for OMAP3 and code flow is clearly
trying to set mpu_clk for OMAP4 for which this dpll_mpu_ck is available.

Signed-off-by: Jarkko Nikula <jhnikula@gmail.com>
---
 arch/arm/mach-omap2/omap2plus-cpufreq.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-omap2/omap2plus-cpufreq.c b/arch/arm/mach-omap2/omap2plus-cpufreq.c
index 8d472f6..d53ce23 100644
--- a/arch/arm/mach-omap2/omap2plus-cpufreq.c
+++ b/arch/arm/mach-omap2/omap2plus-cpufreq.c
@@ -161,7 +161,7 @@ static int __cpuinit omap_cpu_init(struct cpufreq_policy *policy)
 		mpu_clk = clk_get(NULL, "virt_prcm_set");
 	else if (cpu_is_omap34xx())
 		mpu_clk = clk_get(NULL, "dpll1_ck");
-	else if (cpu_is_omap34xx())
+	else if (cpu_is_omap44xx())
 		mpu_clk = clk_get(NULL, "dpll_mpu_ck");
 
 	if (IS_ERR(mpu_clk))
-- 
1.7.10.4

